docs/asciidoc/modules/ROOT/pages/background-operations/apoc-load-directory-async.adoc

```diff
@@ -0,0 +1,16 @@
+The following procedures allow list, remove and add triggers.
+Each trigger consists of a listener observing one or more folders
+which will trigger the execution of a custom cypher query via the following procedures:
```

Comment:
I think the last half sentence is a bit hard to understand. Maybe something like this is better?

> The `apoc.load.directory.async` procedures are used for managing triggers. Each trigger consists of a listener observing one or more folders which will trigger the execution of a custom cypher query. The following procedures can be used to add, remove and list triggers: